posts/feed_bot/galaxy_europe/2024-07-29-ask-us-anything-report.md

```diff
@@ -0,0 +1,53 @@
+---
+media:
+- mastodon-eu-freiburg
+- matrix-eu-announce
+- bluesky-galaxyproject
+- linkedin-galaxyproject
+mentions:
+  mastodon-eu-freiburg:
+  - galaxyproject@mstdn.science
+  bluesky-galaxyproject:
+  - galaxyproject.bsky.social
+  matrix-eu-announce:
+  - bgruening:matrix.org
+hashtags:
+  mastodon-eu-freiburg:
+  - UseGalaxy
+  - GalaxyProject
+  - UniFreiburg
+  - usegalaxy
+  - EOSC
+  - EuroScienceGateway
+  bluesky-galaxyproject:
+  - UseGalaxy
+  - GalaxyProject
+  - UniFreiburg
+  - usegalaxy
+  - EOSC
+  - EuroScienceGateway
+  linkedin-galaxyproject:
+  - UseGalaxy
+  - GalaxyProject
+  - UniFreiburg
+  - usegalaxy
+  - EOSC
+  - EuroScienceGateway
+---
+📝 New blog post Released!
+
+🚀 **First “Ask Us Anything about Galaxy” live Q\&A session was successful**
+
+
+Serving and engaging with the Galaxy user community is one of the key pillars of the Galaxy project. Galaxy beginners and advanced users have many questions around Galaxy concerning data analysis, tools, workflows or the training material. For the very first time, the Galaxy\-EU team offered a live webinar to Galaxy users and gave them the chance to ask questions there and meet the people behind Galaxy\-EU.
+
+
+🌟 **Questions from beginners and advanced users**
+
+
+More than 30 scientists followed our invitation and filled this one hour with interesting questions which were answered by experts of our team. For all of us it was a well received and great experience and we plan to offer online live Q\&A sessions every 2\-3 months. The next one will be offered in September 2024\. We will announce them on the [Events page](https://galaxyproject.org/events).
+
+
+![User Qustions](https://galaxyproject.org/news/2024-07-29-ask-us-anything-report/questions_1.png)
```

Comment:
It looks like the images in [RSS feed of Galaxy news](https://galaxyproject.org/eu/feed.atom) doesn't have the right link.
```suggestion
![User Qustions](https://galaxyproject.org/assets/static/questions_1.175c05c.0caec253b2ef2e0beaac0cb1f447d02c.png)
```

Reply:
Do you know why this is happening?

Reply:
Ok, we need to fix then the Hub. Maybe you can do that or please ask Alireza to have a look.